go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/urfave/cli/v2 v2.3.0
 	golang.org/x/sys v0.0.0-20210330210617-4fbd30eecc44 // indirect
+	golang.org/x/tools v0.1.0
```

Comment:
What is this used for?

Reply:
These are the tools mentioned in `tools.go`.

Reply:
Can you add a comment for this import as it is not actually used in the code.